AwsCryptographicMaterialProviders/runtimes/python/src/aws_cryptographic_materialproviders/smithygenerated/aws_cryptography_keystore/client.py

```diff
@@ -0,0 +1,462 @@
+        """Get a particular version of a Branch Key from the Key Store.
+
+        :param input: Inputs for getting a version of a Branch Key.
+        """
+        return asyncio.run(
+            self._execute_operation(
+                input=input,
+                plugins=[],
+                serialize=_serialize_get_branch_key_version,
+                deserialize=_deserialize_get_branch_key_version,
+                config=self._config,
+                operation_name="GetBranchKeyVersion",
+            )
+        )
+
+    def get_beacon_key(self, input: GetBeaconKeyInput) -> GetBeaconKeyOutput:
+        """Get a Beacon Key from the Key Store.
+
+        :param input: Inputs for getting a Beacon Key
+        """
+        return asyncio.run(
+            self._execute_operation(
+                input=input,
+                plugins=[],
+                serialize=_serialize_get_beacon_key,
+                deserialize=_deserialize_get_beacon_key,
+                config=self._config,
+                operation_name="GetBeaconKey",
+            )
+        )
+
+    async def _execute_operation(
+        self,
+        input: Input,
+        plugins: list[Plugin],
+        serialize: Callable[[Input, Config], Awaitable[DafnyRequest]],
+        deserialize: Callable[[DafnyResponse, Config], Awaitable[Output]],
+        config: Config,
+        operation_name: str,
+    ) -> Output:
+        try:
+            return await self._handle_execution(
+                input, plugins, serialize, deserialize, config, operation_name
+            )
+        except Exception as e:
+            # Make sure every exception that we throw is an instance of ServiceError so
+            # customers can reliably catch everything we throw.
+            if not isinstance(e, ServiceError):
+                raise ServiceError(e) from e
+            raise e
+
+    async def _handle_execution(
+        self,
+        input: Input,
+        plugins: list[Plugin],
+        serialize: Callable[[Input, Config], Awaitable[DafnyRequest]],
+        deserialize: Callable[[DafnyResponse, Config], Awaitable[Output]],
+        config: Config,
+        operation_name: str,
+    ) -> Output:
+        context: InterceptorContext[Input, None, None, None] = InterceptorContext(
+            request=input,
+            response=None,
+            transport_request=None,
+            transport_response=None,
+        )
+        _client_interceptors = config.interceptors
+        client_interceptors = cast(
+            list[Interceptor[Input, Output, DafnyRequest, DafnyResponse]],
+            _client_interceptors,
+        )
+        interceptors = client_interceptors
+
+        try:
+            # Step 1a: Invoke read_before_execution on client-level interceptors
+            for interceptor in client_interceptors:
+                interceptor.read_before_execution(context)
+
+            # Step 1b: Run operation-level plugins
+            # config = deepcopy(config)
```

Comment:
nit: Commented code.

Reply:
Removed

Reply:
I guess this wasn't actually removed...
I'll address this in a follow-up PR